packages/environment/test/develop.js

```diff
@@ -0,0 +1,67 @@
+const assert = require("chai").assert;
+var fs = require("fs-extra");
+var glob = require("glob");
+const { default: Box } = require("@truffle/box");
+const { Environment } = require("@truffle/environment");
+const sinon = require("sinon");
+
+describe("Environment develop", function () {
+  let config;
+
+  const expectedNetwork = {
+    port: 8545,
+    network_id: 1342,
+    gas: 90000
+  };
+
+  const testOptions = {
+    port: 5678,
+    network_id: expectedNetwork.network_id,
+    gas: 7000
+  };
+
+  before("Create a sandbox", async function () {
```

Comment:
You probably want to use `beforeEach` and `afterEach` here. `before` and `after` only get run once while the others run before and after each `it` block.

Reply:
Even though in this case you only have one test so it doesn't matter.

Reply:
change to beforeEach and afterEach